<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Password_Panel</class>
 <widget class="QDialog" name="Password_Panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>141</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Mật khẩu mới</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Mật khẩu cũ</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Nhập lại mật khẩu mới</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>201</width>
     <height>82</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="0">
     <widget class="QLineEdit" name="lineEdit_RewriteNewPassword"/>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEdit_NewPassword"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit_OldPassword"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>43</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_ShowRewriteNewPassword">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/show.png</normaloff>resources/show.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_ShowOldPassword">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/show.png</normaloff>resources/show.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_ShowNewPassword">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/show.png</normaloff>resources/show.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lable_PasswordStatus">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>195</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_Ok">
      <property name="text">
       <string>OK</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/check-mark.png</normaloff>resources/check-mark.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_Cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/stop.png</normaloff>resources/stop.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
